igcollect/ping.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+# this script is used to collect ping data to evaluate availability and latency
```

Comment:
How about using a typo checker before committing to save time for both of us and reduce noise here.

Reply:
typo checker allready run 😅